Kwartracker/Scenes/Wallet Page/WalletPageView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  WalletPageView.swift
+//  Kwartracker
+//
+//  Created by Leah Joy Ylaya on 4/20/21.
+//
+
+import SwiftUI
+
+struct WalletPageView: View {
+    @EnvironmentObject var store: AppStore
+    @State private var margin: CGFloat = 30
+    @State private var buttonToggle: Bool = false
+    private var separator: UIColor = Asset.Colors.spindleGrey.color
```

Comment:
Better change this variable to a `let` instead of `var` if the variable cannot be changed. 🤔

Reply:
Apply it also in lines 15 and 16 when possible